File: Arrays/Java/Sum_equals_to_Sum.java
class Solution { 
    public int findPairs(int arr[],int n) { 
        //code here.
        HashSet<Integer> hs = new HashSet<>();
        for (int i=0; i<n-1; i++) {
            for(int j=i+1; j<n; j++) {
                if (hs.contains(arr[i]+arr[j])) {
                    return 1;
                }
                else {
                    hs.add(arr[i]+arr[j]);
               }
            }
        }
        return 0;
    }
}

File: Solutions/Python/Sum_equals_to_Sum.py
class Solution:
    def findPairs(self, arr, n): 
        #code here.  
        hs = {}
        for i in range(n-1):
            for j in range(i+1,n):
                s = arr[i] + arr[j]
                if s in hs:
                    return 1
                else:
                    hs[s] = [arr[i], arr[j]]
        return 0
